.gitlab-ci.yml

```diff
@@ -278,13 +286,14 @@ build/cuda92/gcc/all/release/shared:
 # cuda 10.0 and friends
 # Make sure that our jobs run when using self-installed
 # third-party HWLOC.
-build/cuda100/gcc/all/debug/shared:
+build/cuda100/mvapich2/gcc/all/debug/shared:
```

Comment:
I think you should have a consistent scheme, either you update the names everywhere or nowhere? I don't mind whichever way as long as it's consistent.

Reply:
My reasoning was that these were the job names and hence they reflect what the job is doing. If there is nompi, I left the name out. I guess I could rename jobs that don't build mpi to `build.../nompi/...`. Would that be better ?